site/content/blogs/release-v120.en.md

```diff
@@ -163,6 +163,43 @@ command to create a pipeline [`manifest.yml`](../docs/manifest/pipeline.en.md) f
     ✔ Successfully deployed pipeline: env-pipeline
     ```
 
+## Autoscaling Cooldown Support
+A small addition to our service manifests: the ability to configure autoscaling cooldown periods.
+For `Load Balanced`, `Backend`, and `Worker` Services, you can now configure this for their corresponding autoscaling fields.
+
+To configure the cooldown field, your service needs some extra fields to allow the scaling to begin.
+First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target which the service will scale to meet.
```

Comment:
I wonder if we can prune any of the text to make it a bit more terse, what do you think of something like?
> You can now configure the autoscaling fields under [`range`](../docs/manifest/lb-web-service.en.md#count-rage) to have custom cooldown periods.
> Previously, each scaling metric such as `cpu_percentage` has a set in cooldown of X secs and out cooldown of Y seconds. Now you can set a global cooldown period:
>  <your yaml>
> Alternatively, you can set individual cooldowns: